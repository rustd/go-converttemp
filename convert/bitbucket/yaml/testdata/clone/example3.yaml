pipelines:
  default:
    - step:
        runs-on:
          - self.hosted
          - linux
        clone:
          lfs: true # See the lfs property documentation prior to enabling
          depth: 2
          skip-ssl-verify: true
        script:
          - ls -R $BITBUCKET_CLONE_DIR
