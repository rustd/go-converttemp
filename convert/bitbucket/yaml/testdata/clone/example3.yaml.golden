{
    "pipelines": {
        "default": [
            {
                "step": {
                    "clone": {
                        "depth": 2,
                        "lfs": true,
                        "skip-ssl-verify": true
                    },
                    "runs-on": [
                        "self.hosted",
                        "linux"
                    ],
                    "script": [
                        "ls -R $BITBUCKET_CLONE_DIR"
                    ]
                }
            }
        ]
    }
}